setup.py

```diff
@@ -53,15 +53,15 @@
         "Development Status :: 3 - Alpha",
         "Intended Audience :: Developers",
         "Intended Audience :: Science/Research",
-        "License :: OSI Approved :: MIT License",
+        "License :: OSI Approved :: Apache v2.0",
         "Programming Language :: Python :: 3.7",
         "Topic :: Scientific/Engineering :: Artificial Intelligence",
         ],
     keywords="",
     author="Flax team",
     author_email="flax-dev@google.com",
     url="https://github.com/google/flax",
-    license="Apache",
+    license="Apache v2.0",
```

Comment:
We can delete this line per [python docs](https://docs.python.org/3.7/distutils/setupscript.html#installing-additional-files).

> The `license` field is a text indicating the license covering the package where the license is not a selection from the “License” Trove classifiers. See the `Classifier` field. Notice that there’s a `licence` distribution option which is deprecated but still acts as an alias for license.